### PR TITLE
Updates on dashboard

### PR DESCRIPTION
**Pull request by asubedy**
Signed-off-by: asubedy <frexpe@pm.me>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Review by warunicorn19 — APPROVED** [review]
LGTM

File: docs/assets/js/toggle.js [hunk 1]
```diff
@@ -48,3 +48,22 @@ window.addEventListener("resize",function(){
...
+  if(loadGenerator.style["display"]=="block"){
+    loadGenerator.style["display"]= "none"
+    iconDown.classList.replace("fa-angle-down","fa-angle-right")
+  }
+  else{
+    loadGenerator.style["display"]="block"
+    iconDown.classList.replace("fa-angle-right","fa-angle-down")
+  }
+  
+  console.log("clicked")
```
Comment (Abhishek-kumar09) [inline docs/assets/js/toggle.js:68]:
```suggestion
```

File: docs/_sass/layout/_dashboard.scss [hunk 2]
```diff
@@ -128,6 +125,10 @@ body{
 	#content.active {
 		margin-left: 0rem;
 	}
+	.load-generators{
+		display: block;
+
```
Comment (Abhishek-kumar09) [inline docs/_sass/layout/_dashboard.scss:130]:
```suggestion
```

File: docs/_layouts/dashboardPage.html [hunk 3]
```diff
@@ -36,17 +36,20 @@
...
-
+			<div id = "selectMeshModal" class="selectServiceMesh">
+			<button onclick=serviceMeshClick() id="selectMeshButton" class="display-5 text-black">Select Service Mesh  <i class="chevron fa fa-angle-right mr-2"></i> </button>
+			<div class = "inputMeshBig">
 			<!-- List service mesh -->
-				<form class="load-generators">
+				<form class="load-generators" style="display: none;">
 					{%- for service_mesh in site.data.service_meshes.service_meshes  -%}
-					<div class="form-check form-check-inline">
+					<div class="form-check" style>
```
Comment (Abhishek-kumar09) [inline docs/_layouts/dashboardPage.html:45]: style doesn't have value assigned

Comment (asubedy) [reply docs/_layouts/dashboardPage.html:45]: Oops ok I'll remove it

**Pull request merged**